File: 2020_ITE1014_2020002542/6-2/4.c
#include <stdio.h>

int main(void)
{
	int a, b, c;
	scanf("%d %d %d", &a, &b, &c);

	int maxNum=a, minNum=a;

	if(a<c)
	{maxNum=c;
		if(c<b)
		{maxNum=b;}
		else if(b<a)
		{minNum=b;}
	}
	else if(a<b)
	{maxNum=b;
		if(b<c)
		{maxNum=c;}
		else if(c<a)
		{minNum=c;}
	}
	else if(a>b)
	{minNum=b;
		if(b>c)
		{minNum=c;}
	}
	
	printf("max: %d \n", maxNum);
	printf("min: %d \n", minNum);

	return 0;
}

File: 2020_ITE1014_2020002542/6-2/4.py
#6-2 4.py
integers=input().split()
integers[0]=int(integers[0])
integers[1]=int(integers[1])
integers[2]=int(integers[2])

maxNum=integers[0]
minNum=integers[0]

if integers[0]<integers[2]:
    maxNum=integers[2]
    if integers[2]<integers[1]:
        maxNum=integers[1]
    elif integers[1]<integers[0]:
        minNum=integers[1]
    
elif integers[0]<integers[1]:
    maxNum=integers[1]
    if integers[1]<integers[2]:
        maxNum=integers[2]
    elif integers[2]<integers[0]:
        minNum=integers[2]

elif integers[0]>integers[1]:
    minNum=integers[1]
    if integers[1]>integers[2]:
        minNum=integers[2]
        
print("min: "+str(minNum))
print("max: "+str(maxNum))
